infra/cluster-api/cluster-packet-template.yaml

```
@@ -0,0 +1,202 @@
+kind: KubeadmControlPlane
+apiVersion: controlplane.cluster.x-k8s.io/v1alpha3
+metadata:
+  name: "${CLUSTER_NAME}-control-plane"
+spec:
+  version: ${KUBERNETES_VERSION}
+  replicas: ${CONTROL_PLANE_MACHINE_COUNT}
+  infrastructureTemplate:
+    apiVersion: infrastructure.cluster.x-k8s.io/v1alpha3
+    kind: PacketMachineTemplate
+    name: "${CLUSTER_NAME}-control-plane"
+  kubeadmConfigSpec:
+    initConfiguration:
+      nodeRegistration:
+        kubeletExtraArgs:
+          cloud-provider: external
+    clusterConfiguration:
+      apiServer:
+        extraArgs:
+          cloud-provider: external
+          audit-policy-file: /etc/kubernetes/pki/audit-policy.yaml
+          audit-log-path: "-"
+          audit-webhook-config-file: /etc/kubernetes/pki/audit-sink.yaml
+      controllerManager:
+        extraArgs:
+          cloud-provider: external
+    joinConfiguration:
+      nodeRegistration:
+        kubeletExtraArgs:
+          cloud-provider: external
+    postKubeadmCommands:
+    - |
+        cat <<EOF >> /etc/network/interfaces
+        auto lo:0
+        iface lo:0 inet static
+          address {{ .controlPlaneEndpoint }}
+          netmask 255.255.255.255
+        EOF
+    - systemctl restart networking
+    - 'kubectl --kubeconfig /etc/kubernetes/admin.conf create secret generic -n kube-system packet-cloud-config --from-literal=cloud-sa.json=''{"apiKey": "{{ .apiKey }}","projectID": "${PROJECT_ID}", "eipTag": "cluster-api-provider-packet:cluster-id:${CLUSTER_NAME}"}'''
+    - kubectl apply --kubeconfig /etc/kubernetes/admin.conf -f https://github.com/packethost/packet-ccm/releases/download/v1.1.0/deployment.yaml
+    - kubectl apply --kubeconfig /etc/kubernetes/admin.conf -f https://github.com/packethost/csi-packet/raw/master/deploy/kubernetes/setup.yaml
+    - kubectl apply --kubeconfig /etc/kubernetes/admin.conf -f https://github.com/packethost/csi-packet/raw/master/deploy/kubernetes/node.yaml
+    - kubectl apply --kubeconfig /etc/kubernetes/admin.conf -f https://github.com/packethost/csi-packet/raw/master/deploy/kubernetes/controller.yaml
+    - kubectl apply --kubeconfig /etc/kubernetes/admin.conf -f "https://cloud.weave.works/k8s/net?k8s-version=$(kubectl version | base64 | tr -d '\n')?IPALLOC_RANGE=172.26.0.0/16"
+    preKubeadmCommands:
+    - mkdir -p /etc/kubernetes/pki
+    - |
+        cat <<EOF >> /etc/kubernetes/pki/audit-policy.yaml
+        apiVersion: audit.k8s.io/v1
+        kind: Policy
+        rules:
+          - level: RequestResponse
+        EOF
+    - |
+        cat <<EOF >> /etc/kubernetes/pki/audit-sink.yaml
+        apiVersion: v1
+        kind: Config
+        clusters:
+          - cluster:
+              server: http://10.96.96.96:9900/events
+              name: auditsink-cluster
```

I believe this is indented too far
```suggestion
            name: auditsink-cluster
```

---

Thank you! Indenting is easy to miss